scripts/migrate_duplicate_external_accounts.py

```diff
@@ -18,6 +18,7 @@
 logger = logging.getLogger(__name__)
 
 def creds_are_valid(ea_id):
+    logger.warn('Validating credentials for externalaccount {}'.format(ea_id))
```

Comment:
This doesn't need to be a warning; DropBox / GitHub don't roll creds. That said, nbd